pkg/util/kube/pvc_pv.go

```diff
@@ -360,6 +360,7 @@ func MakePodPVCAttachment(volumeName string, volumeMode *corev1api.PersistentVol
 		volumeMounts = []corev1api.VolumeMount{{
 			Name:      volumeName,
 			MountPath: volumePath,
+			ReadOnly:  true,
```

Comment:
I think this should not be hardcoded, `MakePodPVCAttachment` is a shared function and is also used by `generic restore`

Reply:
This function is shared with restore in pkg/exposer/generic_restore.go. 

This will cause restores to fail due to readOnly filesystems for PVC in volumeMode: Filesystem. 

@Lyndon-Li readOnly as parameter will fix this.

Reply:
Done, updated the PR.